iceoryx_examples/waitset_on_c/README.md

```diff
@@ -1,15 +1,19 @@
-# WaitSet C binding
+# WaitSet in C
```

Comment:
@ithier @elBoberido @MatthiasKillat I noticed that we are not consistent with the naming when it comes to C stuff, sometimes it's `*_on_c` in other places it's `*_in_c`. Is it ok from your side if I rename all occurrences to `*_in_c` in a follow-up PR?

Reply:
Yep please do. I think we encountered this earlier too but perhaps there are some occurrences that slipped through. 
They should be changed to `in_c` as this is the correct wording.